experiments/mrmc_experiment/run_mrmc_experiment.py

```diff
@@ -432,19 +491,61 @@ def main(
         run_configs = run_configs[:max_runs]
         if verbose:
             print(f"Throw out all but --max_runs={max_runs} run_configs.")
-    if not dry_run:
+    if distributed:
         if verbose:
-            print(f"Start executing {len(run_configs)} mrmc runs.")
-        results = run_batch(run_configs, verbose)
-        results_dir = save_results(results, results_dir, config)
+            print(
+                f"{len(run_configs)} runs will be distributed over "
+                f"{num_processes} processes."
+            )
+    if not dry_run:
+        if distributed:
+            runner = parallel_runner.ParallelRunner(
+                experiment_mainfile_path=__file__,
+                final_results_dir=_get_results_dir(
+                    results_dir, config["experiment_name"]
+                ),
+                num_processes=num_processes,
+                use_slurm=use_slurm,
+                random_seed=None,  # not needed for reproducibility.
+                scratch_dir=scratch_dir,
+                verbose=verbose,
+            )
+            if verbose:
+                print(f"Start executing {len(run_configs)} mrmc runs.")
+            results = runner.execute_runs(run_configs)
+        else:
+            if verbose:
+                print(f"Start executing {len(run_configs)} mrmc runs.")
+            results = run_batch(run_configs, verbose)
+            print("got results")
+        results_dir = save_results(results, results_dir, config, only_csv)
         if verbose:
             print(f"Saved results to {results_dir}")
     elif verbose:
         print("Terminate without executing runs because --dry_run is set.")
 
 
+def validate_args(args: argparse.Namespace, parser: argparse.ArgumentParser):
```

Comment:
I missed this args check at first -- it might be helpful to move this to below the args definitions, which will be helpful in remembering to update them if args change.